src/cplus_plugin/lib/reports/metrics.py

```diff
@@ -412,3 +415,52 @@ def createExpressionContext(self) -> QgsExpressionContext:
         :rtype: QgsExpressionContext
         """
         return create_metrics_expression_context()
+
+
+def calculate_activity_pwl_impact(activity_id: str, number_jobs: float) -> float:
+    """Calculates the PWL impact an activity.
+
+    It sums up the result of the number of jobs multiplied
+    by the area of each NCS pathway that constitutes the
+    activity.
+
+    :param activity_id: The ID of the specific activity.
+    :type activity_id: str
+
+    :param number_jobs: Number of jobs for the activity.
+    :type number_jobs: float
+
+    :returns: Returns the total pwl impact of the activity, or -1.0
+    if the activity does not exist or if found, lacks pathways
+    or if the area of all pathways could not be computed.
+    :rtype: float
+    """
+    activity = settings_manager.get_activity(activity_id)
+    if activity is None:
+        return -1.0
+
+    if len(activity.pathways) == 0:
```

Comment:
You can combine these 2 ifs
```
if activity is None or len(activity.pathways) == 0:
    return -1.0
```

I think they're still readable. Is it perhaps because of lint check, that these ifs are split?